api/desecapi/tests/test_domains.py

```diff
@@ -296,12 +296,6 @@ def test_create_domain_other_parent(self):
         self.assertStatus(response, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.data['name'][0].code, 'name_unavailable')
 
-    def test_create_domain_atomicity(self):
-        name = self.random_domain_name()
-        with self.assertPdnsRequests(self.request_pdns_zone_create_422()):
-            self.client.post(self.reverse('v1:domain-list'), {'name': name})
-            self.assertFalse(Domain.objects.filter(name=name).exists())
-
```

Comment:
why remove this?

Reply:
Because pdns errors are not exclusively treated as server errors, i.e. Django's DB transaction will be rolled back, so it seemed silly to keep it. But yeah, why not: I re-added an updated version of it.